test/WebJobs.Script.Tests.Integration/App.config

```diff
@@ -1,112 +1,112 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
```

Comment:
I can exclude them, but I'd like to see if you get the same behavior from VS on a nuget package update